ucs/database.h

```diff
@@ -46,7 +46,16 @@ class Database : public DBcore {
 	int FindCharacter(const char *CharacterName);
 	bool VerifyMailKey(std::string CharacterName, int IPAddress, std::string MailKey);
 	bool GetVariable(const char* varname, char* varvalue, uint16 varvalue_len);
+	void LoadBlockChannels();
 	bool LoadChatChannels();
+	bool IsOnChannelBlockList(std::string channelName);
+	void AddToChannelBlockList(std::string channelName);
+	void RemoveFromChannelBlockList(std::string channelName);
```

Comment:
These block list methods really should just live within the ChannelList class and have them mutate the private member `m_blocked_channels`

Reply:
Updated